src/app/tests/suites/certification/Test_TC_FLW_2_1.yaml

```diff
@@ -42,19 +42,20 @@ tests:
               minValue: 0
               maxValue: 65533
 
+    #Skipping in CI due to SDK issue: https://github.com/project-chip/connectedhomeip/issues/27543
```

Comment:
This is easier to fix than to work around, fwiw: https://github.com/project-chip/connectedhomeip/pull/29355

Reply:
Updated.